### DISCUSSION TITLE
Existing User’s dashboard is not loading after Kimai migration from version 1.9 to 2.12.0 in docker environment

### DISCUSSION THREAD

**Discussion by sksundar1985** [kimai/kimai#4820] [Docker]
### Describe the problem

**When:** Existing users when either clicks on the Dashboard or Weekly Hours under Time Tracking menu.

**Note:** It’s working perfectly if I  create a new user. 

**Exiting User's Dashboard settings as follow in kimai version 1.9**

                  Display: layout: fixed
                  Initial calendar view: Month
                  Initial view after login: Mytimes
-------------------------------------------
**Error in browser:**
	URL: http://192.168.1.101/en/quick_entry

	Error:
	500 Something is wrong
	A critical error occurred, please try again. You may have found a software problem, please contact your administrator if the problem persists. 

------------------------------------------

**Docker container logs:**
kimai_1  | [2024-03-12T08:23:13.935941+00:00] request.INFO: Matched route "quick_entry". {"route":"quick_entry","route_parameters":{"_route":"quick_entry","begin":null,"_controller":"App\\Controller\\QuickEntryController::quickEntry","_locale":"en"},"request_uri":"http://192.168.1.101/en/quick_entry","method":"GET"} {"channel":"request"}
kimai_1  | [2024-03-12T08:23:14.041926+00:00] request.CRITICAL: Uncaught PHP Exception Error: "Call to a member function getId() on null" at QuickEntryController.php line 81 {"exception":"[object] (Error(code: 0): Call to a member function getId() on null at /opt/kimai/src/Controller/QuickEntryController.php:81)"} {"channel":"request"}
nginx_1  | 192.168.1.210 - - [12/Mar/2024:08:23:14 +0000] "GET /en/quick_entry HTTP/1.1" 500 1589 "http://192.168.1.101/en/timesheet/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:78.0) Gecko/20100101 Firefox/78.0" "-"
kimai_1  | 172.29.0.4 -  12/Mar/2024:08:23:13 +0000 "GET /index.php" 500
nginx_1  | 192.168.1.210 - - [12/Mar/2024:08:23:14 +0000] "GET /favicon.ico HTTP/1.1" 200 15086 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:78.0) Gecko/20100101 Firefox/78.0" "-"

---
kimai_1  | [2024-03-12T08:24:36.881973+00:00] request.INFO: Matched route "dashboard". {"route":"dashboard","route_parameters":{"_route":"dashboard","_controller":"App\\Controller\\DashboardController::index","_locale":"en"},"request_uri":"http://192.168.1.101/en/dashboard/","method":"GET"} {"channel":"request"}
kimai_1  | [2024-03-12T08:24:36.996417+00:00] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "An exception has been thrown during the rendering of a template ("Call to a member function getId() on null")." at widget-paginatedworkingtimechart.html.twig line 30 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"Call to a member function getId() on null\"). at /opt/kimai/templates/widget/widget-paginatedworkingtimechart.html.twig:30)\n[previous exception] [object] (Error(code: 0): Call to a member function getId() on null at /opt/kimai/src/Widget/DataProvider/DailyWorkingTimeChartProvider.php:127)"} {"channel":"request"}
nginx_1  | 192.168.1.210 - - [12/Mar/2024:08:24:37 +0000] "GET /en/dashboard/ HTTP/1.1" 500 1589 "http://192.168.1.101/en/timesheet/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:78.0) Gecko/20100101 Firefox/78.0" "-"
kimai_1  | 172.29.0.4 -  12/Mar/2024:08:24:36 +0000 "GET /index.php" 500

-------------------------------------------
	

### Describe your setup and add your Docker compose file (redact your credentials)

OS Ubuntu 22.04 LTS
migrated from version 1.9 to 2.12.0 in docker environment
Docker Version : Docker version 24.0.5, build 24.0.5-0ubuntu1~22.04.1

**Comment by kevinpapst** [comment]
1.9 to 2.12 ? That was a brave move!
The errors sound like a cache problem.

**Comment by sksundar1985** [comment]
> 1.9 to 2.12 ? That was a brave move! The errors sound like a cache problem.

I did that, in the kimai container,  also tested in different browser in private mode but no luck. Kindly advise is there any other way to fix this. 
bin/console kimai:reload --env=prod
rm -r var/cache/prod/*
